include/Simulator.hpp

```diff
@@ -102,6 +102,10 @@ class Simulator {
         return binary;
     }
 
+    void setTolerance(const dd::fp tolerance) {
+        dd->cn.setTolerance(tolerance);
+    }
```

Comment:
Shouldn't there be a method `getTolerance` as well?

Reply:
Done in https://github.com/cda-tum/ddsim/pull/202/commits/8096c50e477942e9f5a6996ba33537ff4f13b71f